awx/sso/backends.py

```diff
@@ -342,7 +342,7 @@ def _update_m2m_from_groups(ldap_user, opts, remove=True):
     if opts is None:
         return None
     elif not opts:
-        pass
+        return pass
```

Comment:
Just one more minor fix, we need to just pass here instead of calling return. Other than that this is good to go!

Reply:
Of course.